utils/generate_types/overrides.d.ts

```diff
@@ -34,6 +34,8 @@ export interface Page {
   evaluateHandle<R, Arg>(pageFunction: PageFunction<Arg, R>, arg: Arg): Promise<SmartHandle<R>>;
   evaluateHandle<R>(pageFunction: PageFunction<void, R>, arg?: any): Promise<SmartHandle<R>>;
 
+  addInitScript<Arg>(script: PageFunction<Arg, void> | { path?: string, content?: string }, arg?: Arg): Promise<void>;
```

Comment:
```suggestion
  addInitScript<Arg>(script: PageFunction<Arg, any> | { path?: string, content?: string }, arg?: Arg): Promise<void>;
```

This way you will avoid issues where init function returns something. Otherwise it would be a breaking change, as seen by our own tests.